### PR TITLE
Leaderboard C. Header only

### PR DESCRIPTION
**Pull request by jasonsandlin**

### CODE REVIEW

File: Include/xsapi-c/leaderboard_c.h [hunk 1]
```diff
@@ -0,0 +1,383 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+
+typedef enum XblLeaderboardStatType
```
Comment (jplafonta) [inline Include/xsapi-c/leaderboard_c.h]: Enum description if there is one?

File: Include/xsapi-c/leaderboard_c.h [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+/// <param name="hasNext">Return value. True if there are more results to retrieve, false otherwise.</param>
+/// <returns>HRESULT return code for this API operation.</returns>
+STDAPI XblLeaderboardResultHasNext(
+    _In_ xbl_leaderboard_result_handle resultHandle,
+    _Out_ bool* hasNext
+    ) XBL_NOEXCEPT;
+
+/// <summary>
+/// Gets the result of next page of the leaderboard for a player of the specified title.
+/// To get the result, call XblLeaderboardResultGetNextResult inside the AsyncBlock callback
```
Comment (jplafonta) [inline Include/xsapi-c/leaderboard_c.h]: >XblLeaderboardResultGetNextResult [](start = 28, length = 33)

Where is this function?

File: Include/xsapi-c/leaderboard_c.h [hunk 3]
```diff
@@ -0,0 +1,383 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+
+typedef enum XblLeaderboardStatType
+{
+    /// <summary>Unsigned 64 bit integer.</summary>
```
Comment (rakhad) [inline Include/xsapi-c/leaderboard_c.h]: >. [](start = 40, length = 1)

Be consistent with your periods

File: Include/xsapi-c/leaderboard_c.h [hunk 4]
```diff
@@ -0,0 +1,383 @@
...
+    XblLeaderboardSortOrder_Ascending,
+
+    /// <summary> 
+    /// Sorting the leaderboard highest to lowest
+    /// </summary>
+    XblLeaderboardSortOrder_Descending
+} XblLeaderboardSortOrder;
+
+/// <summary> 
+/// The order to sort the leaderboard in
```
Comment (rakhad) [inline Include/xsapi-c/leaderboard_c.h]: >he order to sort the leaderboard in [](start = 5, length = 35)

Incorrect summary

**Pull request closed without merging**